GPII-3590: reorganize and update docs

mrtyler opened the pull request:
I did some reorganization of the docs in preparation for sending them to new users.

I tried to make targeted commits, but big refactors like this can be hard to understand by looking at diffs -- especially for documentation.

You can review the rendered markdown here:
https://github.com/mrtyler/gpii-infra/tree/gpii-3590-docs

The big ideas here are:
* Make "Getting Started" path for new users clearer and more concise
* Extract some areas into separate documents linked from the main READMEs: one-time setup steps, how to contact ops, principles
* Copy relevant FAQ items from AWS to GCP

I suspect I'll find more things to change as I train new users on GCP, hence the WIP label. However I think there's enough here that an early look might be useful for reviewers.

stepanstipl commented on common/README.md:
```diff
@@ -73,13 +73,14 @@ From the root of the repository:
 cp -r john $USER
 #(edit $USER/terraform.tfvars)
   project_name = "dev-$USER"
+  project_owner = "$USER@RtF"
```
Worth to include the `user:` prefix here? ;)

mrtyler replied:
Yes :)

natarajaya commented:
LGTM!

mrtyler commented:
I thought I had put the work-in-progress label on this PR but now I can't find it.

Anyway, I probably should have merged this earlier, but I did the initial GCP training today and added some things we discovered during that process. I think this is ready for a last review before merging, if anyone is so inclined.

I added a parking lot item about running `update_exekube`.

stepanstipl commented:
> I thought I had put the work-in-progress label on this PR but now I can't find it.

:) <img width="321" alt="screenshot 2019-02-20 at 12 43 14" src="https://user-images.githubusercontent.com/3027227/53089531-21d58880-350d-11e9-963a-2a24b09501e2.png">

LGTM
